README.md

```diff
@@ -40,22 +45,19 @@ et al, which specialise in preparing your spatial data for visualisation. Rather
 choice of tool to you the user, as we want GeoVista to remain as flexible and open-ended as possible to the entire
 Scientific Python community.
 
-Simply put, "*GeoVista is to PyVista*", as "*Cartopy is to Matplotlib*". Well, that's the aspiration.
+Simply put, "*GeoVista is to PyVista*", as "*[Cartopy](https://scitools.org.uk/cartopy/docs/latest/) is to
+[Matplotlib](https://matplotlib.org/stable/index.html)*". Well, that's the aspiration.
 
 ## Installation
 
 GeoVista is available on both [conda-forge](https://anaconda.org/conda-forge/geovista) and [PyPI](https://pypi.org/project/geovista/).
 
-We recommend using [mamba](https://github.com/mamba-org/mamba) to install GeoVista 👍
+We recommend using [conda](https://docs.conda.io/projects/conda/en/latest/index.html) to install GeoVista 👍
 
-### Mamba
+### Conda
 
 GeoVista is available on [conda-forge](https://anaconda.org/conda-forge/geovista), and can be easily installed with
-[mamba](https://github.com/mamba-org/mamba):
-```shell
-mamba install -c conda-forge geovista
-```
-or alternatively with [conda](https://docs.conda.io/projects/conda/en/latest/index.html):
+[conda](https://docs.conda.io/projects/conda/en/latest/index.html):
```

Comment:
I'd genuinely prefer to recommend and advertise `mamba`, it's still a far superior experience to `conda` :wink:

Reply:
But let's just go with `conda` rather than my personal preference 👍

Reply:
> I'd genuinely prefer to recommend and advertise `mamba`, it's still a far superior experience to `conda` 😉

Hey if that's what you think then I'm missing some information. I was basing this entirely on the recent inclusion of the Mamba solver within Conda. Please tell me more!

Reply:
For me the hands down winner is parallel downloads of packages ... but honestly, let's just go with `conda`, that's cool.

TBH I'd rather suggest [pixi](https://github.com/prefix-dev/pixi) 😉 

But that's a bridge too far perhaps ... nevertheless, it is officially amazing 🤩